model/ftn/ww3_bound.ftn

```diff
@@ -411,6 +427,11 @@
 !
             READ(200+IP,'(A1,A10,A1,2F7.2,F10.1,F7.2,F6.1,F7.2,F6.1)') &
               space,buoyname,space,LATS(IP),LONS(IP),depth,U10,Udir,Curr,Currdir
+!/RTD
+!/RTD            ! Rotated coordinates are scaled in range 0 - 360
```

Comment:
Would this always be the case if the parent grid wasn't global?

Does a similar coordinate shift need to be applied in ww3_bounc.ftn?

Reply:
> Would this always be the case if the parent grid wasn't global?

It doesn't matter, as the w3iobc module handles [-180,  180] or [0, 360] ranges when reading in the BCs. The alignment here is to ensure we can match the model BC points in _this_ routine (which will be [0,360] after de-rotation) with the input spectra locations.

> Does a similar coordinate shift need to be applied in ww3_bounc.ftn?

No - ww3_bounc has a DIST_SPHERE routine that is used to calculate the distance between points. I guess it could be included in ww3_bound, but I considered it out of scope for this PR.

Reply:
Thanks for the confirmation - I'm happy with this section of code in that case